content/applications/services/field_service/product_management.rst

```diff
@@ -1,10 +1,30 @@
+==================
+Product management
+==================
+
+When field service workers perform on-site, they commonly use products for the service. Odoo’s
+**Field Service** app allows them to record the products they’re using. Doing so keeps your
+inventory up-to-date in real-time and automatically adds these products to the invoice.
+
+Add products to a task
 ======================
```

Comment:
- This might be a question for @zulu-odoo since this was her suggestion, but she mentioned the term "product catalog" in the discord discussion. Is this was is commonly used to refer to this feature? Should we use/mention it somewhere in here?
- I think that you need to enable the Time and Material Invoicing setting to be able to use this feature so, depending on Zuza's answer to my question above, we might want to do something like this:

Product catalog
============

To enable the product catalog, go to :menuselection:`Field Service --> Configuration --> Settings` and enable the :guilabel:`Time and Material Invoicing` feature.

To add products to a task, proceed as follows:

(then keep your steps below)